upgrade/tests/manage_venv/test_get_compatible_upgrade_versions.py

```diff
@@ -0,0 +1,61 @@
+from pathlib import Path
+
+from upgrade.scripts.find_compatible_versions import get_compatible_version
+from upgrade.scripts.requirements import to_requirements_obj
+
+
+def test_get_compatible_version_where_current_installed_version_is_2_0_0_and_compatible_upgrade_version_is_2_0_1_expect_compatible_upgrade_version_returned(
```

Comment:
Maybe shorten this name by just saying current_is_2_0_0_and_compatible_is_2_0_1 or something like that? So that the name can fit in one line.